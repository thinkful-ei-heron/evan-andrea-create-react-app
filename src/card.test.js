import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

describe('<Card />', () => {
    it('renders without crashing', () => {
        const div= document.createElement('div');
        ReactDOM.render(<Card />, div);
        
        ReactDOM.unmountComponentAtNode(div);
});
    it('renders UI as expected', () => {
        const tree= renderer.create(<Card title='cats' content='dogs' />).toJSON();
        
        expect(tree).toMatchSnapshot(); 
    });
});