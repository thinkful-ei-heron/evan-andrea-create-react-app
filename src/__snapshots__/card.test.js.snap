// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    cats
  </h3>
  <p>
    dogs
  </p>
</div>
`;
